src/main/scad/yavuz-demirhan/basket-burr/basketBurr.scad

```diff
@@ -0,0 +1,57 @@
+include <puzzlecad.scad>
+
+// Global settings
+$burr_scale = 10;
+$plate_width = 100;
+$burr_inset = 0.05;
+$joint_inset = 0.08;
+$burr_bevel = 1.0;
+$unit_beveled = false;
+
+*basket();
+*holder();
+*pieces();
+
+module basket() {
+    render(convexity = 2)
+    burr_piece([
+        "xxxxxx|xxxxxx|x....x|......|......|......",
+        "xxxxxx|xxxx{connect=fy+}xx|x....x|......|......|......",
```

Comment:
Just noticed this - why is the basket on its side? Shouldn't it lay flat so that it can be printed without supports? Then the male connectors could go on the basket, with (oriented) female connectors on the handle, avoiding the use of supports entirely.

Reply:
I copied the X text from PWBP, so the orientation is the same with source. I would like to introduce a simplest way to build a model by PWBP's source, so I didn't change it